### DISCUSSION TITLE
How can I get the JSON data from the event tracker?

### DISCUSSION THREAD

**Discussion by ocoke** [umami-software/umami#1723] [Q&A]
I saw that the `event_data` could be a JSON object, so I want to use this to track web performance.

![image](https://user-images.githubusercontent.com/71591824/210161458-18ad9893-d8dc-4f9c-890f-85dd64fcf670.png)

the code is:

```
let times = {}
umami.trackEvent('Timing Data', times);
```

and the request query is:

![image](https://user-images.githubusercontent.com/71591824/210161486-dada0e28-1acb-427a-82ef-be64912ec457.png)

but on the umami dashboard, I only found this:

![image](https://user-images.githubusercontent.com/71591824/210161495-630f3f41-fb46-4e9e-bf7d-6e976210e692.png)

I can't find any other data here.

and also I don't know how to use the `query event data`, thanks.

![image](https://user-images.githubusercontent.com/71591824/210161519-215df63b-c118-443a-be7f-ba6ad9d27839.png)

**Comment by briancao** [comment]
Querying event_data assumes the user knows what field they're trying to query.

Try adding 'DOMInteractiveTime' with a type, 'SUM' to get the total amount of interactive time across the date range.

**Reply by ocoke** [reply]
sorry, but the dashboard return `0`.

![image](https://user-images.githubusercontent.com/71591824/210203422-aa17b92b-c3ae-4f9f-9006-05c57290f38e.png)

in the database:

![image](https://user-images.githubusercontent.com/71591824/210203477-ec4a904d-93ba-4ffa-904f-d6c436287361.png)

**Comment by ocoke** [comment]
however, I found some errors in my console.

![image](https://user-images.githubusercontent.com/71591824/210203613-adcbdb45-0e63-4c65-96af-dcaba6b569af.png)

and I think that server has returned the data successfully.

![image](https://user-images.githubusercontent.com/71591824/210203652-2d3e1a53-7411-437b-a166-59bb16dde9be.png)

**Reply by briancao** [reply]
Let me see if I can reproduce this issue.  It may be a bug.

**Reply by goodevilgenius** [reply]
I've found that `sum` doesn't seem to work. But `count` does.

Just a guess, but maybe `sum` is checking all events, even those that don't have the particular field.

**Reply by ocoke** [reply]
> Let me see if I can reproduce this issue. It may be a bug.

thanks.

**Comment by goodevilgenius** [comment]
Is there no way to show the values for each of the available fields, or, at the very least, get a list of the available fields?

It seems like having to remember exactly what fields I've used for everything is going to be less useful.

**Reply by briancao** [reply]
No plans on providing the keys to collected JSON data at this time.
